### PR TITLE
Fixes #32656 - Add manageBookmarks to pf4 bookmarks

### PR DESCRIPTION
**Pull request by MariaAga**
Duplicating https://github.com/theforeman/foreman/pull/8444 to pf4

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarkItems.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+  Spinner,
+} from '@patternfly/react-core';
 import { sprintf, translate as __ } from '../../../common/I18n';
 import { STATUS } from '../../../constants';
 import DocumentationUrl from '../DocumentationLink';
 
-export const actionItems = ({ canCreate, setModalOpen, documentationUrl }) => (
-  <DropdownGroup key="group 1">
-    {canCreate && (
+export const actionItems = ({ canCreate, setModalOpen }) =>
```
Comment (tbrisker) [inline webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarkItems.js]: Should this be `addBookmark` now that it's the only action here?

**Review by Ron-Lavi — DISMISSED** [review]
Tested via #8743 and works great! thanks @MariaAga !

non-blocking: can you remove the snapshot tests?

**Comment by MariaAga** [comment]
Thanks @laviro! I rather keep this a duplicate of the pf3 pr, and remove snapshots in a different one

**Review by tbrisker — COMMENTED** [review]
Looks good @MariaAga ! one question inline

File: webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarkItems.js (hunk 1 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarkItems.js]: Yeah, renamed to `addBookmarkItem` so it wont be confused with an action that will add a bookmark

**Review by tbrisker — APPROVED** [review]
Thanks @MariaAga and @laviro ! will merge once tests finish.

**Comment by MariaAga** [comment]
@laviro, added https://github.com/theforeman/foreman/pull/8832 to remove snapshots

**Pull request merged**